CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
+## [17.2.2]
+
+This is another maintenance release to address the invalid Java Runtime Environment (JRE) on 
+staging an App Engine standard app ([#1316](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1316))
+issue:
+
+> Unable to stage app: Cannot get the System Java Compiler. Please use a JDK, not a JRE.
```

Comment:
If you look at our other sections, we have a "Fixed" header where we list fixes.It has the issue referenced at the end.